clients/r2cbe/events_client.go

```diff
@@ -0,0 +1,81 @@
+package r2cbe
```

Comment:
This is super cryptic. :slightly_smiling_face: Does it make sense to use something like `chaindata`? Then we could do `chaindata.NewDownloader()` or `chaindata.NewClient()`. Or maybe `chaincache`.